docs/guides/bootstrap_github_ssh.md

```diff
@@ -0,0 +1,116 @@
+---
+subcategory: ""
+page_title: "Bootstrap with GitHub"
+description: |-
+	Install Flux and synchronize with GitHub.
+---
+
+# Bootstrap with GitHub
+
+This guide will walk through how to install Flux into a Kubernetes cluster and configure if to synchronize from a GitHub repository. Begin with creating a GitHub repository, in this guide it will be named `fleet-infra`, which will be used by Flux.
```

Comment:
```suggestion
This guide will walk through how to install Flux into a Kubernetes cluster and configure it to synchronize from a GitHub repository. Begin with creating a GitHub repository, in this guide it will be named `fleet-infra`, which will be used by Flux.
```